tests/test_visualization.py

```diff
@@ -218,8 +219,13 @@ def test_visualization_calls_render_correctly_with_contour(
         solver = new_solver_session_with_exhaust_case_and_data
         TGraphicsWindow.show_graphics = lambda win_id: None
         contour = Contour(
-            solver=solver, field="temperature", surfaces=["in1", "in2", "in3"]
+            solver=solver,
+            field=VariableCatalog.TEMPERATURE,
```

Comment:
The underlying nomenclature is different for file sessions so is worth an extra test.

Reply:
The  FileSession will be implemented as a separate PR.
Issue: https://github.com/ansys/pyfluent-visualization/issues/542